kv/mock_test.go

```diff
@@ -44,7 +44,6 @@ func (s testMockSuite) TestInterface(c *C) {
 		mock.GetOption(23)
 	}
 	transaction.StartTS()
-	transaction.DelOption(23)
```

Comment:
why remove it?

Reply:
don't have `DelOption` now. And following code does not rely on it.